app/templates/direct-award/view-project.html

```diff
@@ -94,50 +80,50 @@
-            "custom_body_list": [
-              {"type": "text", "text": ("Contracts worth over £10,000 must be published on <a href=\"https://www.gov.uk/contracts-finder\" target=\"_blank\" rel=\"external noopener noreferrer\"
-                data-analytics=\"trackEvent\"
-                data-analytics-category=\"Direct Award\"
-                data-analytics-action=\"External Link\"
-                >Contracts Finder</a>.")|safe},
-              {"type": "box", "style": "inactive", "label": "Can't start yet"} if not project.downloadedAt else {}
-            ]
-          },
           {
             "body": "Complete the Customer Benefits Record form",
             "custom_body_list": [
-              {"type": "text", "text": ''.join(["Fill in the Crown Commercial Service’s <a href=\"", framework_urls.customer_benefits_record_form_url|e, "\" target=\"_blank\" rel=\"external noopener noreferrer\"
-                data-analytics=\"trackEvent\"
-                data-analytics-category=\"Direct Award\"
-                data-analytics-action=\"External Link\"
-                >", framework['name'], " Customer Benefits Record form</a>."])},
-              {"type": "text", "text": ''.join(["Email a copy to <a href=\"mailto:", framework_urls.customer_benefits_record_form_email|e, "\" target=\"_top\"
-                  data-analytics=\"trackEvent\"
-                  data-analytics-category=\"Direct Award\"
-                  data-analytics-action=\"External Link\"
-                  >", customer_benefits_record_form_email|e, "</a>."])},
-              {"type": "box", "style": "inactive", "label": "Can't start yet"} if not project.downloadedAt else {}
+              {"type": "text", "text":
+                ''.join(["You must fill in the <a href=\"", framework_urls.customer_benefits_record_form_url|e, "\"
+                                                 target=\"_blank\" rel=\"external noopener noreferrer\"
+                                                 data-analytics=\"trackEvent\"
+                                                 data-analytics-category=\"Direct Award\"
+                                                 data-analytics-action=\"External Link\"
+                                                >Customer Benefits Record form</a>
+                and email it to the Crown Commercial Service at <a href=\"mailto:", framework_urls.customer_benefits_record_form_email|e, "\"
+                                                                   target=\"_top\"
+                                                                   data-analytics=\"trackEvent\"
+                                                                   data-analytics-category=\"Direct Award\"
+                                                                   data-analytics-action=\"External Link\"
+                                                                  >", customer_benefits_record_form_email|replace("-", "&#8209;"|safe), "</a>"]
```

Comment:
Is this missing the `framework_urls.` or are we declaring `customer_benefits_record_form_email` somewhere else in the context? 

Also, I'd probably split this string up for readability, seeing as we're joining it anyway.

```
''.join([
    "You must fill in the",
    " <a href=link with all that analytics stuff>Customer Benefits Record form</a>",
    " and email it to CCS at ",
    " <a href=...>",
    framework_urls.customer_benefits_record_form_email|replace(...),
    "</a>"
])
```